examples/FluentUI.Demo.Shared/wwwroot/css/site.css

```diff
@@ -343,10 +343,6 @@ code {
         max-width: 100%;
     }
 
-    .navmenu-main-button {
-        display: none;
-    }
-
     .menu-icon {
```

Comment:
This was preventing the collapse icon from appearing on mobile phones, which is where we needed it most.